session/session.go

```diff
@@ -2764,25 +2772,38 @@ func splitAndScatterTable(store kv.Storage, tableIDs []int64) {
 	}
 }
 
-// InitDDLJobTables is to create tidb_ddl_job, tidb_ddl_reorg and tidb_ddl_history.
-func InitDDLJobTables(store kv.Storage) error {
+// InitDDLJobTables is to create tidb_ddl_job, tidb_ddl_reorg and tidb_ddl_history, or tidb_ddl_backfill and tidb_ddl_backfill_history.
+func InitDDLJobTables(store kv.Storage, targetVer string) error {
+	targetTables := DDLJobTablesVer1
+	if targetVer == meta.DDLTableVersion3 {
+		if !ddl.IsDistReorgEnable() {
```

Comment:
Since v6.6 I expected， that distributed ddl parallel framework will be always enabled. so I think later version tidb_ddl_backfill and tidb_ddl_backfill_history, these two tables will be always in TiDB.

Reply:
So is this PR going to stay or? We update it after distributed ddl parallel framework always enable.